automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/methodconfigs/ImportMethodConfigModal.scala

```diff
@@ -6,10 +6,12 @@ import org.broadinstitute.dsde.firecloud.component._
 import org.broadinstitute.dsde.firecloud.page.MethodTable
 import org.openqa.selenium.WebDriver
 
-class ImportMethodConfigModal(implicit webDriver: WebDriver) extends Modal("import-method-configuration-modal") {
+class ImportMethodConfigModal(implicit webDriver: WebDriver) extends OKCancelModal("import-method-configuration-modal") {
   private val chooseConfigFromRepoButton = Button("import-from-repo-button" inside this)
   private val copyConfigFromWorkspaceButton = Button("copy-from-workspace-button" inside this)
 
+  override def awaitReady(): Unit = chooseConfigFromRepoButton.awaitVisible()
```

Comment:
This shouldn't be necessary. This modal doesn't do any loading and should be ready as soon as it renders. If this change is actually helping something, we have serious problems with the entire framework.

Reply:
I believe I saw a test failed with a screenshot of this Modal last week, and it led me to this Modal file. I don't have that screenshot saved to show you. Are you okay with ImportMethodConfigModal change but not this?

Reply:
I'm fine with not adding this check now but I can keep an eye on MethodConfigSpec test.